nix/status-go/default.nix

```diff
@@ -51,21 +53,52 @@ let
       };
 
   mobileConfigs = {
-    android = {
+    android = rec {
       name = "android";
-      outputFileName = "status-go-${srcData.shortRev}.aar";
       envVars = [
         "ANDROID_HOME=${androidPkgs.androidsdk}/libexec/android-sdk"
         "ANDROID_NDK_HOME=${androidPkgs.ndk-bundle}/libexec/android-sdk/ndk-bundle"
         "PATH=${makeBinPath [ openjdk ]}:$PATH"
       ];
-      gomobileExtraFlags = [ "-androidapi 18" ];
+      gomobileExtraFlags = [ "-androidapi 23" ];
+      outputFileName = "status-go-${srcData.shortRev}.aar";
+      platforms = {
+        x86 = {
+          linkNimbus = enableNimbus;
+          nimbus = assert enableNimbus; nimbus.wrappers-android.x86;
+          gomobileTarget = "${name}/386";
+          outputFileName = "status-go-${srcData.shortRev}-386.aar";
+        };
+        arm64 = {
+          linkNimbus = enableNimbus;
+          nimbus = assert enableNimbus; nimbus.wrappers-android.arm64;
+          gomobileTarget = "${name}/arm64";
+          outputFileName = "status-go-${srcData.shortRev}-arm64.aar";
+        };
+      };
     };
-    ios = {
+    ios = rec {
       name = "ios";
-      outputFileName = "Statusgo.framework";
       envVars = [];
       gomobileExtraFlags = [ "-iosversion=8.0" ];
+      outputFileName = "Statusgo.framework";
+      platforms = {
+        ios = {
+          linkNimbus = enableNimbus;
+          nimbus = assert false; null; # TODO
+          gomobileTarget = name;
+          inherit outputFileName;
+        };
+      };
+      # platforms = {
+      #   x86 = {
+      #     nimbus = assert enableNimbus; nimbus.wrappers-ios.x86;
+      #     gomobileTarget = "${name}/386";
+      #   };
+      #   arm64 = {
+      #     nimbus = assert enableNimbus; nimbus.wrappers-ios.arm64;
+      #     gomobileTarget = "${name}/arm64";
+      # };
```

Comment:
Why are those commented?

Reply:
This is what the future code will look like for iOS once we provide support for it. It's probably not valuable enough to keep it, since it can be easily adapted from the Android side of things, I'll remove it.

Reply:
I'm fine with leaving it IF you add a comment explaining what it is and what it will be for in the future.